ui/src/app/components/gke-app-configuration-panels/create-cromwell.spec.tsx

```diff
@@ -3,7 +3,7 @@ import '@testing-library/jest-dom';
 import * as React from 'react';
 
 import { DisksApi, WorkspaceAccessLevel } from 'generated/fetch';
-import { AppsApi } from 'generated/fetch/api';
```

Comment:
The export structure of openapi appears to be slightly different. I suspect this allows everything to be imported out of the root.